README.md

```diff
@@ -30,13 +30,10 @@ Contains crates specific to our application. Can depend on libraries located in
 3. Install SSL libraries
    - Ubuntu/Debian: `apt install libssl-dev`
    - Mac ([Homebrew](https://brew.sh/)) `brew install openssl`
-4. `solc` is currently needed to build (will be deprecated). 2 choices:
```

Comment:
:tada:

Reply:
😞 sadly I had to bring it back, the dependencies were currently just moved to the library. 
The task to change solc/docker into a runtime dependency only: https://github.com/comit-network/comit-rs/issues/936